src/appleseed/renderer/kernel/rendering/final/adaptivepixelrenderer.cpp

```diff
@@ -241,7 +241,7 @@ namespace
                     trackers[1].insert(shading_result.m_main[1]);
                     trackers[2].insert(shading_result.m_main[2]);
                 }
-
+                
```

Comment:
Remove trailing spaces.

Reply:
That's what happen when you edit the code directly on github :(

Reply:
Yeah, not recommended especially since it's easy to introduce mistakes that will break compilation :)